### PR TITLE
Analytic api calling

### PR DESCRIPTION
**Pull request by pgvda**

### CODE REVIEW

File: build.rar
Comment (LiniFernando) [inline build.rar:1]: Remove this

File: src/Screens/CategoryAnalytics.js [hunk 1]
```diff
@@ -113,6 +114,8 @@ const CategoryAnalytics = () => {
                 }
             });
 
+            console.log('graph data', response)
```
Comment (LiniFernando) [inline src/Screens/CategoryAnalytics.js:117]: remove console

**Review by LiniFernando — CHANGES_REQUESTED** [review]

File: src/Screens/CategoryAnalytics.js (hunk 1 above)
Comment (pgvda) [reply src/Screens/CategoryAnalytics.js:117]: done

File: build.rar
Comment (pgvda) [reply build.rar:1]: done

**Pull request merged**